azure-pipelines.yml

```diff
@@ -254,3 +254,27 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
+- job: RichCodeNav_Indexing
+  pool:
+    name: NetCorePublic-Pool
```

Comment:
Somewhere here we need to set EnableRichCodeNavigation environment variable to "true" so that log outputs are created.

Reply:
Added EnableRichCodeNavigation as true